content/accounts.md

```diff
@@ -0,0 +1,702 @@
+
+<h3 id="oauth-calling-api">Calling service API for more data</h3>
+
+If your app supports or even requires login with an external service such as Facebook, it's natural to also want to use that service's API to request additional data about that user. For example, you might want to get a list of a Facebook user's photos.
+
+First, you'll need to request the relevant permissions when logging in the user. See the section above for how to pass those options.
+
+Then, you need to get the user's access token. You can find this token in the `Meteor.users` collection under the `services` field. For example, if you wanted to get a particular user's Facebook access token:
+
+```js
+// Given a userId, get the user's Facebook access token
+const user = Meteor.users.findOne(userId);
+const fbAccessToken = user.services.facebook.accessToken;
+```
+
+For more details about the data stored in the user database, read the section below about accessing user data.
+
+Now that you have the access token, you need to actually make a request to the appropriate API. Here you have two options:
+
+1. Use the `http` package to access the service's API directly. You'll probably need to pass the access token from above in a header. For details you'll need to search the API documentation for the service.
+2. Use a package from Atmosphere or NPM that wraps the API into a nice JavaScript interface. For example, if you're trying to load data from Facebook you could use the [fbgraph](https://www.npmjs.com/package/fbgraph) NPM package. Read more about how to use NPM with your app in the [Build System article](build-tool.html#npm).
+
+<h2 id="displaying-user-data">Loading and displaying user data</h2>
+
+Meteor's accounts system, as implemented in `accounts-base`, also includes a database collection and generic functions for getting data about users.
+
+<h3 id="current-user">Currently logged in user</h3>
+
+Once a user is logged into your app with one of the methods described above, it is useful to be able to identify which user is logged in, and get the data provided during the registration process.
+
+<h4 id="current-user-client">On the client: Meteor.userId()</h4>
+
+For code that runs on the client, the global `Meteor.userId()` reactive function will give you the ID of the currently logged in user.
+
+In addition to that core API, there are some helpful shorthand helpers: `Meteor.user()`, which is exactly equal to calling `Meteor.users.findOne(Meteor.userId())`, and the `{{currentUser}}` Blaze helper that returns the value of `Meteor.user()`.
+
```

Comment:
You're right - we need to do some escaping here.

:+1: 39e6add69623f0b7f55b4b0bd20e398bf74707c1